src/Components/Charts/ChooseCoin/SelectCoin.js

```diff
@@ -74,43 +76,44 @@ class DisplayChartNavbar extends React.Component {
       { type: "Volume", id: "volume" },
       { type: "Custom", id: "custom" },
     ],
-    input: "",
   };
+
   handleDuration = (item) => {
     this.state.periodInfo.map((element) => {
       if (element.id === item.id) {
         this.props.alert(element.id);
-        this.setState({ selectedTime: element });
+        this.setState({ selectedTime: element, timeSelect: false });
       }
     });
   };
   handleType = (item) => {
-    this.state.coinInfo.map((element) => {
+    let type = this.state.coinInfo;
+    type.map((element) => {
```

Comment:
you need to use a forEach not map here, map is for creating a new array not for iteration

Reply:
replaced from map to forEach